packages/Adapters/STKMesh/example/CMakeLists.txt

```diff
@@ -25,18 +25,24 @@ TRIBITS_ADD_EXECUTABLE(
   COMM serial mpi
   )
 
+TRIBITS_ADD_EXECUTABLE(
+  STKMeshDeformation
+  SOURCES mesh_deformation.cpp
+  COMM serial mpi
+  )
+
 TRIBITS_COPY_FILES_TO_BINARY_DIR(
   STKInterpolationINP
   SOURCE_FILES input.xml
   SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}
   DEST_DIR ${CMAKE_CURRENT_BINARY_DIR}
-  EXEDEPS STKInterpolation STKRepeatedTransfer STKInlineInterpolation
+  EXEDEPS STKInterpolation STKRepeatedTransfer STKInlineInterpolation STKMeshDeformation
 )
 
 TRIBITS_COPY_FILES_TO_BINARY_DIR(
   STKInterpolationEXO
-  SOURCE_FILES cube_mesh.exo pincell_mesh.exo
+  SOURCE_FILES cube_mesh.exo pincell_mesh.exo mesh_deform_2d.exo
   SOURCE_DIR ${DTK_DATA_DIR}/exodus/
   DEST_DIR ${CMAKE_CURRENT_BINARY_DIR}
-  EXEDEPS STKInterpolation STKRepeatedTransfer STKInlineInterpolation
+  EXEDEPS STKInterpolation STKRepeatedTransfer STKInlineInterpolation STKMeshDeformation
```

Comment:
Yes.  I want it to be clear what example require what xml input/mesh so that it is manageable if we happen to remove or change one of the examples.